notebooks/datasets/nldas_compare2_3.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>Why do the maps default to different zoom levels? Make sure the area of interest is highlighted and it might be useful to have a final paragraph highlighting key areas of difference and what those differences mean.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Both the dataset have different extent and that's the one of the point they want to highlight.</p>